content/blog/blog-post-08.md

```diff
@@ -6,15 +6,16 @@ tags: ["blog"]
 thumbnail: img/blog/2024/kokkos-hpsf.png
 ---
 
-Kokkos is among the six initial technical projets of the newly created High Performance Software Foundation ([HPSF](https://hpsf.org)).
+Kokkos is among the six initial technical projects of the newly created High Performance Software Foundation ([HPSF](https://hpsf.org)).
 HPSF is set up as an umbrella organization within the Linux Foundation to foster collaboration around the development and support of its associated technical projects.
-Conceived through an initial collaboration between the Spack and the Kokkos team, this effort has brought together institutions that want to foster a more collaborative effort of developing and maintaining key open-source projects in the space of high performance computing and related compute intensive areas.
+Conceived through an initial collaboration between the Spack and the Kokkos team, this effort has brought together institutions that want to foster a more cooperative effort of developing and maintaining key open-source projects in the space of high performance computing and related compute intensive areas.
 
 Historically the HPC community has been characterized by a large amount of fragmentation, with many similar or even identical capabilities developed at different institutions.
-The lack of reliable software support funding mechanism in the scientific community led to a reluctance of relying on third party libraries, and the lack of community governance models hindered the establishment of joint multi-institutional development efforts.
+Many projects were reluctant to rely on third party libraries due to the lack of reliable software support funding mechanism in the scientific community.
+Furthermore, the lack of community governance models hindered the establishment of joint multi-institutional development efforts.
 
 For Kokkos the US Department of Energy's (DOE) Exascale Computing Project changed that.
-By providing a framework that included all DOE's national laboratories it was possible to grow the Kokkos team beyond a single institution and establish a multi-laboratory collaborative effort to develop a robust performance portability solution.
+By providing a framework that included all DOE's national laboratories it was possible to grow the Kokkos team beyond a single institution and establish a multi-laboratory effort to develop a robust performance portability solution.
```

Comment:
Suggestion:  "The ECP's re-imagined  multi-US DOE National Laboratory project structure made it possible to grow the Kokkos Ecosystem beyond a single institution, with the upshot of the greater DOE complex having a real stake in performance portability.